ParkingZoneApp/Repository/ParkingSlotRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using ParkingZoneApp.Data;
+using ParkingZoneApp.Models.Entities;
+using ParkingZoneApp.Repository.Interfaces;
+
+namespace ParkingZoneApp.Repository
+{
+    public class ParkingSlotRepository : Repository<ParkingSlot>, IParkingSlotRepository
+    {
+        private readonly ApplicationDbContext _context;
+        public ParkingSlotRepository(ApplicationDbContext context) : base(context)
+        {
+            _context = context;
+        }
+
+        public ICollection<ParkingSlot> GetAllParkingSlots()
```

Comment:
in service we added GetAll() instead of GetAllParkingSlots() thats why we dont need it